pkg/securitypolicy/framework.rego

```diff
@@ -1325,20 +1339,45 @@ fragment_version_is_valid {
     some fragment in data.policy.fragments
     fragment.issuer == input.issuer
     fragment.feed == input.feed
-    semver.compare(data[input.namespace].svn, fragment.minimum_svn) >= 0
+    svn_ok(data[input.namespace].svn, fragment.minimum_svn)
 }
 
 fragment_version_is_valid {
```

Comment:
So apropos of nothing I believe you've found a bug in the error code! The purpose of `fragment_version_is_valid` is to provide a check so that we can write an error when the fragment version is invalid. In order to test whether a fragment is valid we need to find a matching fragment policy object which is in the policy, or which was imported in another fragment (i.e. fragments can themselves contain fragments). The second instance of `fragment_version_is_valid` is supposed to check the fragments of fragments case, but is written incorrectly.